File: BACKEND/RestAPI/src/main/java/com/example/demo/entities/BloodResponse.java
package com.example.demo.entities;


import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString


@Entity
@Table(name ="blood_response")
public class BloodResponse {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int resid;
	
	@ManyToOne
	@JoinColumn(name="brid")
	BloodRequest brid;
	
	byte[] comment;
	String status;
	
}

File: BACKEND/RestAPI_net/Admin/Models/BloodResponse.cs
using System;
using System.Collections.Generic;

namespace Admin.Models;

public partial class BloodResponse
{
    public int Resid { get; set; }

    public int Brid { get; set; }

    public byte[]? Comment { get; set; }

    public string? Status { get; set; }

    public virtual BloodRequest Br { get; set; } = null!;
}
